docs/develop/architecture/records.md

```diff
@@ -96,7 +96,7 @@ removed when they are either published or deleted/discarded.
 Every time a record is created or updated a copy of record is written as a
 record revision. The changes are only tracked on the record versions and only
 happens when a draft is published. This ensures that it is only published
-changes which a tracked.
+changes which are tracked.
```

Comment:
Oof, thanks for identifying how that whole paragraph could use some love! What about something like:

---

Every time a record is created or updated, a copy of the record is written as a *record revision*. The revisions are tracked per record version and only when the corresponding version draft is published. This ensures that only published changes are tracked.

---

? If that's better, add it to your PR and I will merge. Thanks!

Reply:
That reads much clearer to me! I added those changes to this PR.